src/vm/loaderallocator.hpp

```diff
@@ -485,7 +485,6 @@ typedef VPTR(LoaderAllocator) PTR_LoaderAllocator;
 class GlobalLoaderAllocator : public LoaderAllocator
 {
     VPTR_VTABLE_CLASS(GlobalLoaderAllocator, LoaderAllocator)
-    VPTR_UNIQUE_OVERRIDE(VPTRU_LoaderAllocator+1)
```

Comment:
The `VPRT_UNIQUE_OVERRIDE` worked... but later the `VPRT_VTABLE_CLASS` got viral as I had to pass virtual or override info to the underlying macro VPTR_ANY_CLASS_METHODS and was worried about changes on this global macro. Also clang is making an all or nothing: once you are using `override` in a class, you can't use virtual anymore to mark an override... and I really don't want to cleanup all the usage of `VPRT_TABLE_CLASS` with overrides in this PR, as the `override` keyword is barely used in coreclr and definitely not with any `VPRT_VTABLE_CLASS`...

So what should I do? I could add `virtual` modifier at least to `override` methods?